### ISSUE TITLE
where do i connect chroma output to? 

### ISSUE THREAD

**Issue by text2sql** [logspace-ai/langflow#208]
I understand what it does (local vector storage, right?) but can't figure out which dot to connect to, where?

**Comment by ogabrielluiz** [comment]
You should connect it to the VectorStoreInfo node and then connect that one to a VectorStoreAgent.

**Comment by ATL64** [comment]
This worked for me:
<img width="1182" alt="Screen Shot 2023-04-26 at 10 26 22 PM" src="https://user-images.githubusercontent.com/24615574/234706848-6b6301fe-bb0c-4cf5-8fa8-58eacbbca1d8.png">

**Comment by abdallahalhakim77** [comment]
I am trying the same flow as above but as soon as I upload the pdf the entire langflow crashes. Did you or anyone experience this. This is my first attempt!

**Comment by abdallahalhakim77** [comment]
<img width="270" alt="Screen Shot 2023-04-26 at 11 16 56 PM" src="https://user-images.githubusercontent.com/39634000/234775834-1883efcd-4008-4fd5-a02c-8dbd9e72d04e.png">

this is the PDF I am attempting to upload. it is 10MB in size

**Comment by ATL64** [comment]
When running locally I had no issues with file size myself.  But for the GCP version as installed in the README tutorial, I could not upload more than around 800kb PDFs.  The reason I believe is nginx defaulting to a limit of 1MB for POST requests, returning a 413 status code.  I tried to increase that in a config file and restart nginx, and then it allowed me only to upload up to 1.5MB (although I put a much higher value than that).  I did not do much more on this problem. I would suggest to try with smaller PDFs and see if that works, so at least you know it's a matter of file size and not something else.
